### PR TITLE
Triple rollout

### PR DESCRIPTION
**Pull request by travier**
https://github.com/coreos/fedora-coreos-streams/issues/301
https://github.com/coreos/fedora-coreos-streams/issues/302
https://github.com/coreos/fedora-coreos-streams/issues/303

```
stable
    version: 33.20210426.3.0
    start: 2021-05-05 14:00:00+00:00 UTC (in 0:48:19)
           2021-05-05 10:00:00-04:00 Raleigh/New York/Toronto
           2021-05-05 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
next
    version: 34.20210503.1.0
    start: 2021-05-05 14:00:00+00:00 UTC (in 0:48:19)
           2021-05-05 10:00:00-04:00 Raleigh/New York/Toronto
           2021-05-05 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
testing
    version: 34.20210427.2.0
    start: 2021-05-05 14:00:00+00:00 UTC (in 0:48:19)
           2021-05-05 10:00:00-04:00 Raleigh/New York/Toronto
           2021-05-05 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
```

### CODE REVIEW

**Comment by travier** [comment]
Feels like we missed a barrier release for the next stream

File: updates/next.json [hunk 1]
```diff
@@ -30,10 +30,18 @@
     },
     {
       "version": "34.20210427.1.0",
+      "metadata": {
+        "rollout": {
+          "start_percentage": 1.0
+        }
+      }
+    },
```
Comment (jlebon) [inline updates/next.json]: Hmm, any reason we're retaining the last two rollouts? I don't think it hurts, but usually we just edit the `version` field of the previous completed rollout.

**Review by jlebon — APPROVED** [review]
```
next
    version: 34.20210503.1.0
    start: 2021-05-05 14:00:00+00:00 UTC (in 0:47:09)
           2021-05-05 10:00:00-04:00 Raleigh/New York/Toronto
           2021-05-05 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
stable
    version: 33.20210426.3.0
    start: 2021-05-05 14:00:00+00:00 UTC (in 0:47:09)
           2021-05-05 10:00:00-04:00 Raleigh/New York/Toronto
           2021-05-05 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
testing
    version: 34.20210427.2.0
    start: 2021-05-05 14:00:00+00:00 UTC (in 0:47:09)
           2021-05-05 10:00:00-04:00 Raleigh/New York/Toronto
           2021-05-05 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
```

File: updates/next.json (hunk 1 above)
Comment (jlebon) [reply updates/next.json]: Bahh, this is from the outdated diff; ignore.

**Pull request merged**